src/_kw

```diff
@@ -18,10 +18,14 @@ _kw()
     # Set the context for the command
     curcontext="${curcontext%:*:*}:kw-${command}"
     # Narrow the range of words we are looking at to exclude 'kw'
-    (( CURRENT-- ))
+    ((CURRENT--))
```

Comment:
This change doesn't belong to this commit. Please, move it to where this line was introduced the first time.